src/main/java/com/example/employees/service/EmployeeService.java

```diff
@@ -0,0 +1,56 @@
+package com.example.employees.service;
+
+import com.example.employees.model.Employee;
+import com.example.employees.repo.EmployeeRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+@Service
+@AllArgsConstructor
+public class EmployeeService {
+
+    private final EmployeeRepository employeeRepository;
+
+    public Employee findEmployeeByNo(Integer no) {
+        return employeeRepository.findById(no).orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND));
+    }
+
+    public String deleteEmployeeByNo(Integer no) {
+        try {
+            employeeRepository.deleteById(no);
+            return "Deleted employee with ID " + no;
```

Comment:
I believe we can move this response string to presentation layer.
And this method returns nothing (void)